x-pack/plugins/infra/public/apps/metrics_app.tsx

```diff
@@ -29,7 +28,7 @@ export const renderApp = (
 ) => {
   const storage = new Storage(window.localStorage);
 
-  prepareMountElement(element);
+  prepareMountElement(element, 'infraMetricsPage');
```

Comment:
This benefits the functional tests. We previously had a wrapping element in the hierarchy ( a `<ColumnarPage />` I think that added these, but I've removed those. I added it here so it belongs to the top most elements we own.

These get used in the functional tests as a pretty generic "logs / metrics loaded". 

The tests could be reworked to care about the specific page, rather than a more generic wrapper. And then the specific page template strings could be used instead.

This really just pertains to the feature controls / spaces tests. Let me know if you think it's worth the time here.

Reply:
Thanks for the explanation. I think it's fine for now. :+1: